pkgs/by-name/vi/virglrenderer/package.nix

```diff
@@ -11,8 +11,12 @@
   libX11,
   libdrm,
   libgbm,
+  nativeContextSupport ? !stdenv.hostPlatform.isDarwin,
```

Comment:
Are some of these Linux specific or could also work on BSD?

Reply:
I think native context is very linux specific, since it basically forwards the drm uapi. So IMO it should be `nativeContextSupport ? stdenv.hostPlatform.isLinux`.

Venus only depends on vulkan on the host AFAIK, so in theory if there is vulkan support then it could work? (But isn't MacOS Metal only?)

Reply:
Both depend on `libdrm`, which is not available on Darwin.

Reply:
> So IMO it should be `nativeContextSupport ? stdenv.hostPlatform.isLinux`.

That is what I am hinting at.

Reply:
Done, sorry for misunderstanding it first.